lib/core/src/sdk.rs

```diff
@@ -95,11 +101,39 @@ impl LiquidSdk {
         Ok(sdk)
     }
 
+    fn validate_api_key(api_key: &str) -> Result<()> {
+        let pem = [
```

Comment:
I think the certificate is already in DER format (encoded as base64). Perhaps it is better to just decode the base64 into a slice of bytes and then parse is from that format using this? https://docs.rs/x509-parser/latest/x509_parser/fn.parse_x509_certificate.html
Instead of adding the prefix/suffix to adapt it to PEM format.

Reply:
Right, I tried to use that method directly, but will try b64 decoding first. Thanks!